package.json

```diff
@@ -1,12 +1,13 @@
 {
   "name": "playground",
   "private": true,
-  "version": "0.0.0",
+  "version": "0.0.1",
   "scripts": {
```

Comment:
Please synchronize and update README https://github.com/amphitheatre-app/playground#local-development